share/spice/first_robotics_teams/first_robotics_teams.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_first_robotics_teams = function(api_result) {
+
+        if (!api_result || api_result.error || api_result.length === 0 || !api_result.name) {
+            return Spice.failed('first_robotics_teams');
+        }
+        
+        // Fix the displaying of "null" for teams with no motto
+        if (api_result.motto === null) {api_result.motto = 'No motto';}
```

Comment:
Why not remove the empty space for the motto like you did for the moreAt? This would involve building the `record_data` object outside of `Spice.add`.